test/jobs/close_patients_job_test.rb

```diff
@@ -61,6 +61,7 @@ def teardown
     ClosePatientsJob.perform_now
     updated_patient = Patient.find_by(id: patient.id)
     assert_equal(updated_patient.histories.last.history_type, History::HISTORY_TYPES[:record_automatically_closed])
+    assert_histories_contain(patient, 'Monitoree has completed monitoring. Reason: Enrolled more than 14 days after last date of exposure (system)')
```

Comment:
`14 days` here should be a variable right? Didn't we reduce the monitoring period to 10 days?

Reply:
`ADMIN_OPTIONS['monitoring_period_days']` determines the monitoring period so it would be a good idea to use that instead of a hardcoded '14' in this message

Reply:
See latest commit https://github.com/SaraAlert/SaraAlert/pull/1014/commits/8df1fd7cfc3372db37aa9e1772fc1ab1300cf2fd

Reply:
@jkufro the tests also need to use the option to ensure that if it is set to some other value than 14 locally the test won't fail.